### PR TITLE
Polio 88 add translations

### PR DESCRIPTION
**Pull request by quang-le**
## Changes

- Enable Iaso to import translation json files from plugins
- Move polio translations out of iaso, into their own json files
- Replace hard-coded text with formatted messages
- Add missing messages
- Add English translations to polio's `en.json`
- Add English translations to `fr.json` in preparation of [POLIO-94](https://bluesquare.atlassian.net/browse/POLIO-94?atlOrigin=eyJpIjoiYzU4YTA1MDcyNzY4NDJmODlmYjJmZDRjZjEwMmJjYjYiLCJwIjoiaiJ9)
- Left translation jsons unsorted to ease comparison when adding French translations

### CODE REVIEW

File: plugins/polio/js/src/components/ImportLineListDialog.js [hunk 1]
```diff
@@ -60,12 +60,13 @@ const ImportLineListDialog = ({ renderTrigger }) => {
...
-            const message = `Imported successfully ${res.import_result.created} campaign(s)`;
             dispatch(
                 enqueueSnackbar({
                     messageObject: {
                         id: 'successMessage',
-                        defaultMessage: message,
+                        defaultMessage:
+                            'Imported successfully {amount} campaign(s)',
+                        values: { amount: res.import_result.created },
                     },
```
Comment (olethanh) [inline plugins/polio/js/src/components/ImportLineListDialog.js:70]: this is a workaround for a snackbar bug not so I am not sure it will works.

**Review by olethanh — CHANGES_REQUESTED** [review]
blocking for the moment since there is a bug with the snackbar, will try to fix a workaround for it

File: plugins/polio/js/src/components/ImportLineListDialog.js (hunk 1 above)
Comment (quang-le) [reply plugins/polio/js/src/components/ImportLineListDialog.js]: I restored the workaround so the PR can be merged and we can start adding translations in the right place

**Review by olethanh — APPROVED** [review]

**Review by beygorghor — APPROVED** [review]

**Comment by olethanh** [comment]
fixed the conflict, merging

**Pull request merged**